changelogs/unreleased/add-libunwind-submodule.md

```diff
@@ -0,0 +1,8 @@
+## feature/build
+
+* Added bundling of _GNU libunwind_ to support backtrace feature on
+  _AARCH64_ architecture and distributives that don't provide _libunwind_
+  package.
+* Re-enabled backtrace feature for all _RHEL_ distributions by default, except
+  for _AARCH64_ architecture and ancient _GCC_ versions, which lack compiler
+  features required for backtrace (gh-4611).
```

Comment:
Here I'll highlight the same point as in the comment re RPM spec: we should cleanly specify what we offer in our packages. It is not a conditional thing (at least it should not depend on a GCC version on a build host, which is unknown for a user).